STYLE_API_TEMPLATE.md

```diff
@@ -104,9 +132,9 @@ Upon success, the response returns "acknowledged": true
 }
 ```
 
-### Request body
+### Response body
```

Comment:
Remove this section, it is a duplicate (typo?) from the Descriptive section above for POST APIs.

Reply:
Rename this section "Response fields"